package.json

```diff
@@ -42,14 +42,15 @@
     "lint-staged": "^15.2.9",
     "node-fetch": "^3.3.2",
     "prettier": "^3.3.3",
-    "serve": "^14.2.3",
+    "serve": "^6.5.8",
```

Comment:
This doesn't seem right ... 6.5.8 is a few years old?

Reply:
Interesting, I'm assuming npm audit "updated" this 🤔 Fixed!